### PR TITLE
add deployment of Storybook to Github pages

### LINKED ISSUE #70: Deploy Storybook to Github pages
Make sure these boxes are ticked before submitting a new issue:

* [x] I have checked that this is not a duplicate
* [x] I have added a label categorizing this under BLKN | Kiwicom | Icons | Global

### PR DESCRIPTION
**Pull request by Fallion**
Closes #70

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,27 +1,42 @@
...
-#
-# Check https://circleci.com/docs/2.0/language-javascript/ for more details
-#
 version: 2
 jobs:
-  build:
+  test:
     docker:
-      - image: circleci/node:8
+      - image: circleci/node:8.9
```
Comment (jaroslav-kubicek) [inline .circleci/config.yml]: why exacly?

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,27 +1,42 @@
...
 
-      - run: yarn test-ci
+workflows:
+  version: 2
+  test_and_deploy:
+    jobs:
+      - test:
+          filters:
+            branches:
+              ignore:
```
Comment (jaroslav-kubicek) [inline .circleci/config.yml]: this ignore shouldn't be needed since commit contains `ci-skip` in message

**Comment by jaroslav-kubicek** [comment]
We also shouldn't forget about docs update 🙂

File: .circleci/config.yml (hunk 1 above)
Comment (Fallion) [reply .circleci/config.yml]: It's always preferable to specify exact version. Unless we want to always run latest.

File: .circleci/config.yml (hunk 2 above)
Comment (Fallion) [reply .circleci/config.yml]: Still got an email :/

**Comment by Fallion** [comment]
Added docs update to #76

**Pull request merged**